.github/workflows/main.yml

```diff
@@ -9,7 +9,7 @@ on:
   workflow_dispatch:
 
 env:
-  VERSION_PREFIX: "1.0.5"
```

Comment:
Breaking change?

Reply:
There isn't a breaking change since 
- the simple search return type is using a child class of the previous class. If previously users had code like
`ODataValueOfIListOfEntry result = await RepositoryApiClient.SimpleSearchesClient.CreateSimpleSearchOperationAsync(...);`, they would just be using the parent class of `ODataValueContextOfIListOfEntry`. Both `ODataValueOfIListOfEntry` and `ODataValueContextOfIListOfEntry` will work. Just use `ODataValueContextOfIListOfEntry` to get some additional properties
- `FuzzyType` change only occurs when serializing the enum.